evadb/evadb.yml

```diff
@@ -25,7 +25,7 @@ server:
   socket_timeout: 60
 
 experimental:
-  ray: True
+  ray: False
```

Comment:
We need to update the circle ci test setup, after making the default to be False.  Currently we have all tests  with ray enabled except `COVERAGE`. If we want to keep this test setup, we need to modify the circleci configuration accordingly.

Reply:
Yes, by default, let us disable it. For each python version, we could have build with ray disabled and ray enabled (ray installed and configuration variable set to True).